deploy/template.yaml

```diff
@@ -54,15 +58,15 @@ Mappings:
       apiTracingEnabled: false
   EnvironmentVariables: # This is all the environment specific environment variables that don't belong in globals.
     dev:
-      EXAMPLE: 1 # Just here as null values not allowed in tempaltes
+      ISSUER: 'https://review-c.dev.account.gov.uk' # Just here as null values not allowed in tempaltes
```

Comment:
We can remove the comments now these are populated

Reply:
Done